tests/e2e/test_spyre_cb.py

```diff
@@ -95,16 +96,35 @@ def create_random_request(
         request_id: int, num_tokens: int,
         sampling_params: SamplingParams) -> EngineCoreRequest:
 
-    return EngineCoreRequest(request_id=str(request_id),
-                             prompt_token_ids=[request_id] * num_tokens,
-                             mm_inputs=None,
-                             mm_hashes=None,
-                             mm_placeholders=None,
-                             sampling_params=sampling_params,
-                             eos_token_id=None,
-                             arrival_time=0,
-                             lora_request=None,
-                             cache_salt=None)
+    # Temporary until 'data_parallel_rank' parameter makes it to
+    # a release version in vllm
+    if "data_parallel_rank" in [
+            x[0] for x in inspect.getmembers(EngineCoreRequest)
+    ]:
```

Comment:
lil nit that we could do this more concisely like:
```
if "data_parallel_rank":
    extra_kwargs = {"data_arallel_rank": None}
else:
    extra_kwargs = {}

return EngineCoreRequest(
    ...
    **extra_kwargs)
```

but my care amount here is low